### PR TITLE
Feature/unified UI enhancement/mob 1936 implement color pallet based default theme for chat non list attributes

### PR DESCRIPTION
**Pull request by DavDo**
MOB 1936

**Jira issue:**
https://glia.atlassian.net/browse/MOB-1936

### CODE REVIEW

**Review by gugalo — COMMENTED** [review]
I think you have not separated the commits to multiple PRs which I assume is what you wanted to do from the PR names

**Comment by DavDo** [comment]
@gugalo To move forward I need changes from previous PR and so on, thats why I have cherry-picked that commits to new branches. I made `ready for review` only 1-st PR, others are drafts and waiting for 1-st one to merge. this one is also still draft. If you have better suggestion I'd appreciate it.

**Comment by gugalo** [comment]
Could probably make PR merge into the other PR branch instead of the master for now and then later once the other PR is merged change it back to the master. Or just add in PR description which commits should be ignored for review.
But not critical. Can leave it as is. I just thought that that was unintended

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt:42]: It seems to me that this comment should say something like "Default theme for badge", shouldn't it?

**Review by andrews-moc — APPROVED** [review]

**Comment by DavDo** [comment]
> 

I faced this once, and this is the main reason why I ended up from this solution

**Comment by DavDo** [comment]
> > Could probably make PR merge into the other PR branch instead of the master for now and then later once the other PR is merged change it back to the master. Or just add in PR description which commits should be ignored for review. But not critical. Can leave it as is. I just thought that that was unintended
> 
> But keep [this](https://salemove.slack.com/archives/CBJJE330U/p1678353313453439) in mind. 🙂

I faced this once, and this is the main reason why I ended up from this solution

**Comment by DavDo** [comment]
!merge

**Pull request merged**
